src/test/java/com/google/api/codegen/gapic/testdata/php/php_no_path_templates.baseline

```diff
@@ -378,7 +378,7 @@ use PHPUnit\Framework\TestCase;
 use stdClass;
 
 /**
- * @group library
+ * @group example
```

Comment:
How did this change happen?

Reply:
This is so because we stopped using artman_yaml's `api_name` value for this value but generate it from service package name instead. 

The package name for this service is actually `Google\Example\V1`. For example for language service (real one form `googleapis`) is `Google\Cloud\Language\V1`, so it would produce `@group langauge` here (same as the old behavior).

In general I believe the "example" is more correct name here, the fact that it was using "library" before was caused by us reusing same artman_yaml in tests for multiple gapics (which does not correspond to the real world scenario, where each service has its own artman yaml).

I confirmed with @michaelbausor and @dwsupplee that it is ok and is not going to break any tooling.